src/Open3D/Visualization/Rendering/Filament/FilamentView.cpp

```diff
@@ -58,8 +67,23 @@ FilamentView::~FilamentView() {
     engine_.destroy(view_);
 }
 
-void FilamentView::SetDiscardBuffers(const TargetBuffers& buffers)
-{
+void FilamentView::SetMode(Mode mode) {
+    switch (mode) {
+        case Mode::Color:
+            view_->setClearColor(
+                    {clearColor_.x(), clearColor_.y(), clearColor_.z(), 1.f});
+            break;
+        case Mode::Depth:
+            view_->setClearColor(kDepthClearColor);
+            break;
+        case Mode::Normals:
+            break;
```

Comment:
Wouldn't Mode::Normals need some clear color, too? I assume that the shader won't be run for areas that don't have any objects, so they will get whatever clear color there is, which might be something odd (e.g. sky blue) from Mode::Color.

Reply:
I will add code for this mode in my other task for normals rendering